lib/entitlement_info_wrapper.dart

```diff
@@ -175,3 +195,16 @@ enum Store {
   /// For entitlements granted via an unknown store.
   unknownStore
 }
+
+/// Enum of ownership types
+enum OwnershipType {
+  /// For ownership when a user made the purchase themselves
+  purchased,
```

Comment:
These docs look great, but maybe we can reuse the docs in the iOS enum? https://github.com/RevenueCat/purchases-ios/blob/main/Purchases/Public/PurchaseOwnershipType.swift#L20

Reply:
Used the iOS docs and also updated the `purchases-android` ones to match. I included the UNKNOWN because it looks like the linter wants all public fields documented.